app/controllers/home_controller.rb

```diff
@@ -1,11 +1,21 @@
 class HomeController < ApplicationController
-  skip_before_action :authenticate_user!
+  # skip_before_action :authenticate_user!
   def index
-    @users = User.all
+    @users = User.all.order(:name).page(params[:page])
+    @posts = Post.order(:created_at).page(params[:page])
 
     respond_to do |format|
       format.csv { send_data @users.to_csv , filename: "users-#{Date.today}.csv" }
       format.html
     end
   end
+
+  def get_comments
+
+    post = Post.find(params[:id])
```

Comment:
Please handle the case when it'd throw error of `Couldn't fine post with id: ###`